pkg/coscheduling/coscheduling.go

```diff
@@ -228,6 +228,21 @@ func (cs *Coscheduling) Permit(ctx context.Context, state *framework.CycleState,
 		waitTime = 0
 	}
 
+	// We will also request to move the sibling pods back to activeQ.
```

Comment:
I think it is fine.

One might consider doing activation only when numWaitingPods < minAvailable - if group is already permitted maybe there is no need to activate remaining pods? I do not have strong opinions here, but for simplicity I think it is better to always activate pods.

Reply:
> One might consider doing activation only when numWaitingPods < minAvailable - if group is already permitted maybe there is no need to activate remaining pods?

Are you suggesting move the logic to `case core.Wait` only?

Reply:
Possibly, I think it kind of depends how important is to schedule pods above the minAvailable limit.

Reply:
Yup, that makes sense to me.